frontend/src/api/apiFetcher.ts

```diff
@@ -1,6 +1,7 @@
 import { ApiContext } from "./apiContext";
 
-const baseUrl = "https://admin.nykea.org/api";
+const baseUrl = "http://localhost:1338/api"; //LOCAL
+// const baseUrl = "https://admin.nykea.org/api"; //PROD
```

Comment:
Should we keep `baseURL` two versions? or is it better keeping only prod version?

Reply:
Good Catch!! This is supposed to be PROD. Updated to switch to LOCAL only when needed and not supposed to be commiting.